src/topology/bases.lean

```diff
@@ -216,6 +216,11 @@ begin
       image_subset_iff.2 hvu⟩ }
 end
 
+lemma continuous_of_basis_is_open_preimage {β : Type*} [topological_space β]
```

Comment:
If you name it like
```suggestion
protected lemma is_topological_basis.continuous {β : Type*} [topological_space β]
```
then one can call it using `hB.continuous f hf`. Also, one can find it using `#print prefix` command.

Otherwise LGTM
bors d+